tests/acceptance/features/webUIFiles/breadcrumb.feature

```diff
@@ -4,11 +4,12 @@ Feature: access breadcrumb
   So that I can access resources with ease
 
   Background:
-    Given user "Alice" has been created with default attributes
+    Given user "Alice" has been created with default attributes and without skeleton files
 
 
   Scenario: Breadcrumb navigation should not happen on last segment
-    Given user "Alice" has created folder "simple-folder/subfolder"
+    Given user "Alice" has created folder "simple-folder"
+    And user "Alice" has created folder "simple-folder/subfolder"
```

Comment:
IMO
```suggestion
    Given user "Alice" has created folder "simple-folder/subfolder"
```

Reply:
we can not create a subfolder and parent folder at the same time like that.